src/style.css

```diff
@@ -1 +1,33 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+
+  background-image: url(./images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
```

Comment:
You can use just 1 background - https://developer.mozilla.org/ru/docs/Web/CSS/background